tests/integration/test_resolution.py

```
@@ -52,7 +53,14 @@ async def test_action_node_resolution_pipe(url: str) -> None:
             except Exception as e:
                 errors.append(f"Error for node {node.id}: {e}")
 
-    assert total_count > 0, "No nodes found"
+    if total_count <= 0:
+        if obs.screenshot is not None:
```

Ruff (0.11.9)

Undefined name `obs`

(F821)